packages/runtime/container-runtime/src/test/containerRuntime.spec.ts

```diff
@@ -2593,9 +2594,7 @@ describe("Runtime", () => {
 					registryEntries: [],
 					existing: false,
 					requestHandler: undefined,
-					runtimeOptions: {
-						enableGroupedBatching: false,
-					},
+					runtimeOptions: {},
```

Comment:
This and the next one look like behavior changes to me - switching from `false` to the default value `true`

Reply:
I missed that, thanks for pointing it out!